lib/routes/shared/security_pin/backup_phrase/verify_backup_phrase_page.dart

```diff
@@ -171,14 +169,28 @@ class VerifyBackupPhrasePageState extends State<VerifyBackupPhrasePage> {
   }
 
   _selectIndexes() {
-    List list = List.generate(23, (i) => i);
-    list.shuffle();
-    for (var i = 0; i < 3; i++) {
-      _randomlySelectedIndexes.add(list[i]);
-    }
+    var n1 = Random().nextInt(12);
+    var n2 = Random().nextInt(12) + 12;
+    var n3 = Random().nextInt(22);
+    if (n3 >= n1) n3++;
+    if (n3 >= n2) n3++;
+    _randomlySelectedIndexes.addAll([n1, n2, n3]);
     _randomlySelectedIndexes.sort();
   }
 
+  bool hasRange(int start, int end, List numbers) {
```

Comment:
can remove this method now.

Reply:
[Done](https://github.com/breez/breezmobile/pull/180/commits/82ef333d135368f9b757c3608a3ba7cb19fa6c99)